scripts/warnings.sh

```
@@ -0,0 +1,60 @@
+		for v in "${!v_}"; do
+			c="$1-$v"
+			command -v "$c" >/dev/null || continue
+			((n = ${#v}, n = n < 3 ? (3 - n) + k : k))
+			o=$("$c" "-std=$s" "${@:3}" -o "dbs26-$c-$s" 2>&1)
+			w=$(grep -o -- '\[-W[^]][^]]*\]' <<< "$o"|sort -u)
+			[[ -z "$w" ]] || {
+				w="${w//[}"
+				w="${w//]}"
+				printf '\e[32m%s\e[m%*s \e[35m%s\e[m -std=\e[36m%s\e[m%*s' \
+				       "$c" $n '' $("$c" "-std=$s" -xc -E -dM - <<< ''     \
+				                   | grep __STDC_VERSION__ | cut -d_ -f6-) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)